src/nvim/ex_cmds.c

```diff
@@ -4822,8 +4822,8 @@ static void helptags_one(char_u *dir, char_u *ext, char_u *tagfname,
 
   // Find all *.txt files.
   size_t dirlen = STRLEN(dir);
-  STRCPY(NameBuff, dir);
-  STRCAT(NameBuff, "/**/*");
+  STRLCPY(NameBuff, dir, sizeof(NameBuff));
+  STRCAT(NameBuff, "/**/*");  // NOLINT
```

Comment:
I generally agree, but when I tried to do The Right Thing instead of just matching the text `/*`, running `python clint.py src/ex_cmds.c` took longer than 7 minutes (that's when I killed it) opposed to 1 second before the change.

Not sure if there's an easy way to speed this up or if we should go for a simple regexpr (which also means more false-positives):

```py
_RE_PATTERN_OLD_STYLE_COMMENT_LEADER = re.compile(r'''
        (?:
          [^'"]+    # Anything but quote leaders
          |
          "[^"]*"   # Any string enclosed in double quotes
          |
          '[^']*'   # Any string enclosed in single quotes
        )*
        /\*         # Old-style comment leader
        ''', re.VERBOSE)
```